### PR TITLE
Updates security processing console 

### PR DESCRIPTION
**Pull request by Aboshedab**
1- Added missing regulations from the wiki to processing console.
2- Update and adjust descriptions, names, credit fines and time to match the wiki for the processing console.

For the fines, it raises the minimum to what is on the wiki for the most part. The maximum is then changed to be half more, double or following specific instructions of the wiki.

All this is done in communication with Lancer.

https://wiki.aurorastation.org/index.php?title=User:Aboshedab

Wiki is ready for replacement for when the PR is merged.

### CODE REVIEW

**Comment by Aboshedab** [comment]
!review

**Review by mikomyazaki — APPROVED** [review]

File: code/modules/law/laws/low_severity.dm [hunk 1]
```diff
@@ -53,148 +53,148 @@
...
 	max_brig_time = 8
 
-	min_fine = 50
-	max_fine = 300
+	min_fine = 250
+	max_fine = 375
 
 /datum/law/low_severity/radio_misuse
-	name = "i107 - Misuse of Public Radio Channels"
+	name = "i106 - Misuse of Public Radio Channels"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:60]: Dont change the id it is used for db statistics.

File: code/modules/law/laws/low_severity.dm [hunk 2]
```diff
@@ -53,148 +53,148 @@
...
 	desc = "To continually broadcast unimportant, untrue, or insignificant messages on the public radio frequency."
-	id = "i107"
+	id = "i106"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:62]: same here

File: code/modules/law/laws/low_severity.dm [hunk 3]
```diff
@@ -53,148 +53,148 @@
...
+	max_brig_time = 3
 
-	min_fine = 50
-	max_fine = 300
+	min_fine = 150
+	max_fine = 225
 
 /datum/law/low_severity/violating_injunction
-	name = "i108 - Violation of Injunction"
+	name = "i107 - Violation of Injunction"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:71]: same here

File: code/modules/law/laws/low_severity.dm [hunk 4]
```diff
@@ -53,148 +53,148 @@
...
 	desc = " To violate the terms of an injunction made by Security or other legal professions."
-	id = "i108"
+	id = "i107"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:73]: same here

File: code/modules/law/laws/low_severity.dm [hunk 5]
```diff
@@ -53,148 +53,148 @@
...
 	max_brig_time = 10
 
-	min_fine = 0
-	max_fine = 200
+	min_fine = 500
+	max_fine = 750
 
 /datum/law/low_severity/slander_hos
-	name = "i109 - Slandering a Head of Staff"
+	name = "i108 - Slandering a Head of Staff"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:82]: same here

File: code/modules/law/laws/low_severity.dm [hunk 6]
```diff
@@ -53,148 +53,148 @@
...
 	desc = "To directly insult a Head of Staff with no valid complaints."
-	id = "i109"
+	id = "i108"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:84]: same here

File: code/modules/law/laws/low_severity.dm [hunk 7]
```diff
@@ -53,148 +53,148 @@
...
-	min_fine = 50
-	max_fine = 300
+	min_fine = 250
+	max_fine = 375
 
 /datum/law/low_severity/slander
-	name = "i110 - Slander"
-	desc = "To spread false rumors in order to damage someone's reputation."
-	id = "i110"
+	name = "i109 - Slander or Verbal Abuse"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:93]: same here

File: code/modules/law/laws/low_severity.dm [hunk 8]
```diff
@@ -53,148 +53,148 @@
...
+	desc = "To spread false rumors in order to damage someone's reputation, or to use racially offensive language."
+	id = "i109"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:95]: same here

File: code/modules/law/laws/low_severity.dm [hunk 9]
```diff
@@ -53,148 +53,148 @@
...
 	max_brig_time = 5
 
-	min_fine = 50
-	max_fine = 300
+	min_fine = 150
+	max_fine = 250
 
 /datum/law/low_severity/failure_execute
-	name = "i111 - Failure to Execute an Order"
+	name = "i110 - Failure to Execute an Order"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:104]: same here

File: code/modules/law/laws/low_severity.dm [hunk 10]
```diff
@@ -53,148 +53,148 @@
...
 	desc = "To ignore or disregard a superior's valid orders."
-	id = "i111"
+	id = "i110"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:106]: same here

File: code/modules/law/laws/low_severity.dm [hunk 11]
```diff
@@ -53,148 +53,148 @@
...
 	max_brig_time = 15
 
-	min_fine = 50
-	max_fine = 550
+	min_fine = 500
+	max_fine = 750
 
 /datum/law/low_severity/sexual_harassment
-	name = "i112 - Sexual Harassment"
+	name = "i111 - Sexual Harassment"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:115]: same here

File: code/modules/law/laws/low_severity.dm [hunk 12]
```diff
@@ -53,148 +53,148 @@
...
 	desc = "To make unwanted sexual advances or obscene remarks towards another employee."
-	id = "i112"
+	id = "i111"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:117]: same here

File: code/modules/law/laws/low_severity.dm [hunk 13]
```diff
@@ -53,148 +53,148 @@
...
 	min_brig_time = 5
 	max_brig_time = 15
 
 	min_fine = 0
-	max_fine = 200
+	max_fine = 0
 
 /datum/law/low_severity/animal_cruelty
-	name = "i113 - Animal Cruelty"
+	name = "i112 - Animal Cruelty"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:126]: same here

File: code/modules/law/laws/low_severity.dm [hunk 14]
```diff
@@ -53,148 +53,148 @@
...
 	desc = "To inflict unnecessary suffering or harm upon animals with malicious intent."
-	id = "i113"
+	id = "i112"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:128]: same here

File: code/modules/law/laws/low_severity.dm [hunk 15]
```diff
@@ -53,148 +53,148 @@
...
 
 	min_fine = 0
-	max_fine = 200
+	max_fine = 0
 
 /datum/law/low_severity/vandalism
-	name = "i114 - Vandalism"
-	desc = " 	To deliberately damage or deface the station without malicious intent."
-	id = "i114"
+	name = "i113 - Vandalism"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:137]: same here

File: code/modules/law/laws/low_severity.dm [hunk 16]
```diff
@@ -53,148 +53,148 @@
...
+	desc = "To deliberately damage or deface the station without malicious intent. To cause easily repairable damage to an owned IPC, up to and including disabling the chassis."
+	id = "i113"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:139]: same here

File: code/modules/law/laws/low_severity.dm [hunk 17]
```diff
@@ -53,148 +53,148 @@
...
 	max_brig_time = 10
 
-	min_fine = 50
-	max_fine = 500
+	min_fine = 450
+	max_fine = 675
 
 /datum/law/low_severity/threat_murder
-	name = "i115 - Threat of Murder or Serious Injury"
+	name = "i114 - Threat of Murder or Serious Injury"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:148]: same here

File: code/modules/law/laws/low_severity.dm [hunk 18]
```diff
@@ -53,148 +53,148 @@
...
 	desc = "To threaten to kill or seriously injure an employee."
-	id = "i115"
+	id = "i114"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:150]: same here

File: code/modules/law/laws/low_severity.dm [hunk 19]
```diff
@@ -53,148 +53,148 @@
...
 	max_brig_time = 10
 
-	min_fine = 50
-	max_fine = 550
+	min_fine = 500
+	max_fine = 750
 
 /datum/law/low_severity/necrophilia
-	name = "i116 - Disrespect to the Dead"
+	name = "i115 - Disrespect to the Dead"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:159]: same here

File: code/modules/law/laws/low_severity.dm [hunk 20]
```diff
@@ -53,148 +53,148 @@
...
 	desc = "To threaten to kill or seriously injure an employee."
-	id = "i116"
+	id = "i115"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:161]: same here

File: code/modules/law/laws/low_severity.dm [hunk 21]
```diff
@@ -53,148 +53,148 @@
...
 	max_brig_time = 10
 
-	min_fine = 50
-	max_fine = 450
+	min_fine = 400
+	max_fine = 600
 
 /datum/law/low_severity/excessive_detainment
-	name = "i117 - Excessive Use of Force in Detainment"
+	name = "i116 - Excessive Use of Force in Detainment"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:170]: same here

File: code/modules/law/laws/low_severity.dm [hunk 22]
```diff
@@ -53,148 +53,148 @@
...
 	desc = "To use more than the required force to subdue a suspect."
-	id = "i117"
+	id = "i116"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:172]: same here

File: code/modules/law/laws/low_severity.dm [hunk 23]
```diff
@@ -53,148 +53,148 @@
...
 	max_brig_time = 10
 
-	min_fine = 50
-	max_fine = 400
+	min_fine = 350
+	max_fine = 700
 
 /datum/law/low_severity/violation_privacy
-	name = "i118 - Violation of Privacy Laws"
+	name = "i117 - Violation of Privacy Laws"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:181]: same here

File: code/modules/law/laws/low_severity.dm [hunk 24]
```diff
@@ -53,148 +53,148 @@
...
 	desc = "To intentionally leak private aspects of employee records or financial account information to unauthorized personnel."
-	id = "i118"
+	id = "i117"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:183]: same here

File: code/modules/law/laws/low_severity.dm [hunk 25]
```diff
@@ -53,148 +53,148 @@
...
 	max_brig_time = 10
 
-	min_fine = 50
-	max_fine = 550
+	min_fine = 500
+	max_fine = 750
 
 /datum/law/low_severity/illegal_search
-	name = "i119 - Illegal Search"
+	name = "i118 - Illegal Search"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:192]: same here

File: code/modules/law/laws/low_severity.dm [hunk 26]
```diff
@@ -53,148 +53,148 @@
...
 	desc = "To preform a search of an employee or workplace without a search warrant, clearance, or probable cause."
-	id = "i119"
+	id = "i118"
```
Comment (Arrow768) [inline code/modules/law/laws/low_severity.dm:194]: same here

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**